include/stdexec/execution.hpp

```diff
@@ -5736,13 +5736,428 @@ namespace stdexec {
   inline constexpr sync_wait_with_variant_t sync_wait_with_variant{};
 
   namespace __any {
+    template <class T>
+    struct __type{};
```

Comment:
You can use `__mtype` from the `__meta.hpp` header.

Reply:
Oh I have missed that!